app/models/document/document_filter.rb

```diff
@@ -74,8 +74,7 @@ def unpaginated_documents
     documents = Document
     documents = documents.where(block_type: Schema.valid_schemas)
     documents = documents.where(testing_artefact: false) unless Current.user&.is_e2e_user?
-    documents = documents.live
-    documents = documents.joins(:latest_published_edition)
+    documents = documents.joins(:editions).merge(Edition.most_recent_for_document)
```

Comment:
This change isn't like-for-like and will mean the search results will show drafts. I don't mind if that ends up being included in this PR but I just thought I'd point it out.